### PR TITLE
Fixing federation testing docs. Fixes #878

### LINKED ISSUE #878: Update federation docs.
The federation docs aren't updated, doing this rn.

### PR DESCRIPTION
**Pull request by dessalines**

### CODE REVIEW

File: docs/src/contributing_federation_development.md [hunk 1]
```diff
@@ -5,14 +5,12 @@
 If you don't have a local clone of the Lemmy repo yet, just run the following command:
 
 ```bash
-git clone https://github.com/LemmyNet/lemmy -b federation
+git clone https://github.com/LemmyNet/lemmy
 ```
 
 If you already have the Lemmy repo cloned, you need to add a new remote:
```
Comment (Nutomic) [inline docs/src/contributing_federation_development.md:11]: This part isnt needed anymore, as its all in the normal master branch.

File: docs/src/contributing_federation_development.md [hunk 2]
```diff
@@ -26,18 +24,34 @@ You need to have the following packages installed, the Docker service needs to b
...
-username `lemmy_alpha` and `lemmy_beta` respectively, with password `lemmy`.
+The federation test sets up 3 instances:
+
+Instance / Username | Location
+--- | ---
+lemmy_alpha | [127.0.0.1:8540](http://127.0.0.1:8540)
+lemmy_beta | [127.0.0.1:8550](http://127.0.0.1:8550)
+lemmy_gamma | [127.0.0.1:8560](http://127.0.0.1:8560)
+
+You can log into each using the instance name, and `lemmy` as the password, IE (`lemmy_alpha`, `lemmy`).
```
Comment (Nutomic) [inline docs/src/contributing_federation_development.md:39]: Much better :+1:

File: docs/src/contributing_federation_development.md [hunk 3]
```diff
@@ -47,11 +61,12 @@ Follow the normal installation instructions, either with [Ansible](administratio
 ```
```
Comment (Nutomic) [inline docs/src/contributing_federation_development.md:61]: Btw the federation image hasnt been updated in two months. It should work fine with the normal release images, as they contain all the necessary code.

File: docs/src/contributing_federation_development.md (hunk 1 above)
Comment (dessalines) [reply docs/src/contributing_federation_development.md:11]: The plus has it removed :)

Comment (Nutomic) [reply docs/src/contributing_federation_development.md:11]: Not sure what you mean, you are still adding a new remote which isnt needed.

Comment (dessalines) [reply docs/src/contributing_federation_development.md:11]: Ohh okay I see what you mean now, yes I can remove this section.

**Comment by dessalines** [comment]
Is this good to merge @Nutomic ?

**Comment by Nutomic** [comment]
Yep its good :+1: 

Btw I just tried upgrading voyager.lemmy.ml to v0.7.8 as federation should work just fine with the production images. But nothing worked, so I suppose there must be a bug somewhere?

**Comment by dessalines** [comment]
It could be that it tried to run DB migrations that were already there? Or maybe differing columns... Its best to just clear them out and start from scratch.

**Pull request merged**